cocotb/bus.py

```diff
@@ -174,18 +168,16 @@ def __delattr__(self, name):
         return _capture
 
     def sample(self, obj, strict=False):
-        """
-        Sample the values from the bus, assigning them to obj.
+        """Sample the values from the bus, assigning them to ``obj``.
 
         Args:
-            obj (any type) : object with attribute names that match the bus
-                             signals
-
-        Kwargs:
-            strict (bool)  : Check that all signals being sampled are present in obj
+            obj (any type): Object with attribute names that match the bus
```

Comment:
Could remove the `(any type)` as the type specification is optional in the standard anyway.